cmd/airplane/demo/createdb/createdb.go

```diff
@@ -0,0 +1,50 @@
+package createdb
+
+import (
+	"context"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/airplanedev/cli/cmd/airplane/auth/login"
+	"github.com/airplanedev/cli/pkg/cli"
+	"github.com/airplanedev/cli/pkg/logger"
+	"github.com/airplanedev/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+type Config struct {
+	Root *cli.Config
+	Name string
+}
+
+func New(c *cli.Config) *cobra.Command {
+	var cfg = Config{Root: c}
+
+	cmd := &cobra.Command{
+		Use:   "createdb",
```

Comment:
minor but do we typically combine words or do we kebab case? 

```suggestion
		Use:   "create-db",
```

Reply:
TBH I was trying to figure this out - the only multi-word command I could find in our CLI was `apikeys`, which was combined words (though arguments e.g. `--changed-files` are kebab-case). I'm fine with doing this either way?

Reply:
yeah let's do kebab case!